main.js

```diff
@@ -9,6 +9,24 @@ function reverseString(str) {
   return ans;
 }
 
+//by using For Of (where the variable takes the values of a object)
+function reverseStringByForOf(str) {
+  let ans = "";
+  for (let i of str) {
+    ans = i + ans;
+  }
+  return ans;
+}
+
+//by using for loop
+function reverseString(str) {
+  let ans = "";
+  for (let i = 0; i < str.length; i++) {
+    ans = str[i] + ans;
+  }
+  return ans;
+}
+
```

Comment:
Also attempt using the `join` method on strings, in a one-liner

Reply:
Done